picard/ui/options/plugins.py

```diff
@@ -159,7 +158,19 @@ def _populate(self):
 
         self._user_interaction(True)
 
+    def _remove_all(self):
+        self._user_interaction(False)
+        for i, p in self.items.items():
+            idx = self.ui.plugins.indexOfTopLevelItem(i)
+            item = self.ui.plugins.takeTopLevelItem(idx)
+            del item
```

Comment:
I guess that is because item here is a TreeWidgetItem. (also I just used the code from reload_list_of_plugins function)